src/scripts/main.js

```diff
@@ -1,17 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const h1 = document.querySelector('h1');
```

Comment:
your .notification element should be a `div`, not `h1`

Reply:
please fix this comment
```
const body = document.querySelector('body');
const notification = document.createElement('div');
```

Reply:
Still not fixed